inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureEditorMaterial.ts

```diff
@@ -0,0 +1,173 @@
+import { NodeMaterial, InputBlock, NodeMaterialSystemValues, TransformBlock, VertexOutputBlock, TextureBlock, OneMinusBlock, ColorMergerBlock, AnimatedInputBlockTypes, TrigonometryBlockOperations, TrigonometryBlock, FragmentOutputBlock, FragCoordBlock, MultiplyBlock, AddBlock, ModBlock, ClampBlock, LerpBlock, Texture } from 'babylonjs';
+
+export function TextureEditorMaterial(texture : Texture) : NodeMaterial {
+    let nodeMaterial = new NodeMaterial("node");

```

Comment:
Why not using a snippet instead? so we can get rid of this file

Reply:
I will look into that! Thanks

Reply:
Replaced it with a snippet! Thanks for the suggestion!